Process.h

```diff
@@ -49,6 +50,7 @@ typedef enum ProcessField_ {
    TGID = 52,
    PERCENT_NORM_CPU = 53,
    ELAPSED = 54,
+   SCHEDPOL = 55,
    PROC_COMM = 124,
    PROC_EXE = 125,
```

Comment:
Is there any reason you abbreviate `SCHED_POLICY` to `SCHEDPOL`? Because I would like the `SCHED_POLICY` name which is more readable.

Reply:
I think skipping the underscore here to avoid intrusion into the `SCHED_` pseudo-namespace, to avoid confusion where this constant belongs to.

Reply:
@BenBE Good pont about the namespace collision. How about naming it like `SCHEDPOLICY` or `SCHEDULEPOLICY`?

Reply:
`SCHEDULERPOLICY` would be fine by me; though I don't quite care either way.